examples/data_readers.ipynb

```diff
@@ -272,7 +272,7 @@
    "id": "e2285f19-9b34-4484-beaa-79df890b2825",
    "metadata": {},
    "source": [
-    "# A deeper dive into `CSVData`\n",
+    "## A deeper dive into `CSVData`\n",
```

Comment:
This has needed to be changed for a while. The auto-documentation gets messed up if there are two of the largest headings in the same example.